frost-ed25519/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+name = "frost-ed25519"
+edition = "2021"
+# When releasing to crates.io:
+# - Update html_root_url
+# - Update CHANGELOG.md
+# - Create git tag.
+version = "0.1.0"
+authors = ["Deirdre Connolly <durumcrustulum@gmail.com>", "Chelsea Komlo <me@chelseakomlo.com>"]
```

Comment:
Aren't you a co-author, @conradoplg ? :D

Reply:
I guess I am :laughing: Added in 5687e4b25860e